metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -1205,6 +1306,10 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
             )
 
         self.reporter.report_stage_start("explore_metadata")
+        # Extract independent look here, so that explore of this look would get consider in
+        # _make_explore_metadata_events
+        yield from self.extract_independent_looks()
```

Comment:
this should go before the call to `self.reporter.report_stage_start("explore_metadata")`

Reply:
done